### PR TITLE
Details page added, test written, formatting done

### PR DESCRIPTION
**Pull request by CLebby-code**

### CODE REVIEW

**Review by jrlord — CHANGES_REQUESTED** [review]
Can you please also link the company name displayed in the customer list

File: website/views.py
Comment (jrlord) [inline website/views.py:1]: company_details = Company.objects.get(id=pk)
company_instance = Company.objects.get(id=pk)

You're hitting the db twice for exactly the same thing. You only need one.

Comment (jrlord) [inline website/views.py:1]: The customers are fetched in the view, and put as the value for the key 'company_customers' in the template context. Fair enough. But then in the template, it is ignored, and {% for customer in company_instance.customers.all %} is done instead. Either leave the template as it is and sort out the template vars (don't put things you don't need in there), or use the template var in the template.

**Comment by CLebby-code** [comment]
This is done @jrlord

**Review by jrlord — APPROVED** [review]
lgtm

**Pull request merged**